document_hosting/controllers/main.py

```diff
@@ -30,7 +30,7 @@ def get_document(self, oid=-1):
             download=True,
         )
         if status == 304:
-            return Response(status, headers)
+            return Response(status=status, headers=headers)
```

Comment:
good catch! the first argument is `response` ([doc](https://werkzeug.palletsprojects.com/en/2.0.x/wrappers/)). `status` and `headers` are 2nd and 3rd.

Reply:
thx for the doc !